core/src/test/java/org/infinispan/conflict/impl/MultipleCachesDuringConflictResolutionTest.java

```diff
@@ -0,0 +1,55 @@
+package org.infinispan.conflict.impl;
+
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.conflict.MergePolicy;
+import org.infinispan.manager.CacheContainer;
+import org.infinispan.partitionhandling.BasePartitionHandlingTest;
+import org.infinispan.partitionhandling.PartitionHandling;
+import org.infinispan.test.TestingUtil;
+import org.infinispan.test.fwk.TransportFlags;
+
+public class MultipleCachesDuringConflictResolutionTest extends BasePartitionHandlingTest {
```

Comment:
Is the only purpose of this test demonstrating a merge with number of caches > default # of threads in some executor? Comment would be welcome.